tests/PathTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Brick\Http\Tests;
+
+use Brick\Http\Path;
+
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Unit tests for class Path.
+ */
+class PathTest extends TestCase
+{
+    protected $path;
+
+    public function setUp()
+    {
+        $this->path = new Path('/path1/path2');
+    }
+
+    public function testGetParts()
+    {
+        $result = $this->path->getParts();
+
+        $this->assertCount(2, $result);
+        $this->assertSame('path1', $result[0]);
+        $this->assertSame('path2', $result[1]);
+    }
+
+    public function testContains()
+    {
+        $resultTrue = $this->path->contains('path1');
+        $resultFalse = $this->path->contains('path3');
+
+        $this->assertTrue($resultTrue);
+        $this->assertFalse($resultFalse);
```

Comment:
You can merge 2 lines in 1 here: `$this->assertTrue($this->path->contains('path1'));`.